src/pages/PostAJob.js

```diff
@@ -1,24 +1,60 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+import Layout from '../layouts/Layout'
+import { motion } from 'framer-motion'
+import { loadStripe } from '@stripe/stripe-js'
+
+//components
 import PostAJobForm from '../components/form/PostAJobForm'
 import StatusBar from '../components/form/StatusBar'
 import JobTemplate from '../components/JobTemplate'
 import JobPostingConfirmation from '../components/JobPostingConfirmation'
-import { motion } from 'framer-motion'
-import Layout from '../layouts/Layout'
+import TierSelect from '../components/form/TierSelect'
 
+//firebase
 import { db, storage } from '../firebase/firebase'
 import firebase from 'firebase'
 
-const PostAJob = () => {
-  const [status, setStatus] = useState(1)
+const stripePromise = loadStripe(
```

Comment:
`stripePromise` is probably not necessary. Consider just using `loadStripe` with `await` in the code below.

```javascript
const stripe = await loadStripe(
  process.env.REACT_APP_STRIPE_API_KEY
)
```

Reply:
Fixed with latest commit.